lib/services/services_linux.c

```diff
@@ -677,9 +677,19 @@ async_action_complete(mainloop_child_t *p, pid_t pid, int core, int signo,
         parse_exit_reason_from_stderr(op);
 
     } else if (mainloop_child_timeout(p)) {
+        const char *reason = NULL;
+
+        if (op->rsc != NULL) {
+            reason = "Resource agent did not complete in time";
+        } else if (pcmk__str_eq(op->standard, PCMK_RESOURCE_CLASS_STONITH,
+                                pcmk__str_none)) {
+            reason = "Fence agent did not complete in time";
+        } else {
+            reason = "Process did not complete in time";
```

Comment:
"Process" is Kernel/OS terminology. "Resource agent" & "Fence agent" is pacemaker/cluster terminology.
Do we have something generic that matches better?
No really good idea on my side though. As it is called service library maybe "Service" but that is probably
more cool for developers and debugging than for actual users. "Job", "Task" ... all not really catchy ...

Reply:
IIRC, the "Process" branch is used within pacemaker only for alert agents. However external code could theoretically use the library for any file execution. If it's not a resource agent or fence agent, then it is a generic OS process. I suppose we could add a flag to indicate it's an alert agent, and log a separate message for that, but I think the existing log context should be sufficient for alerts.

Reply:
Guess it isn't too important - just wanted to see if we can come up with something from resource-manager terminology.
Maybe "Process/Agent" is something that doesn't require adding code to gather more info.